### PR TITLE
I've changed the function of the log in page, that everytime the user…

### PR DESCRIPTION
**Pull request by sulfur132**
… is trying to log in, the button will load, so that the user will not be confused if the user is logged in or not.

### CODE REVIEW

File: frontend/src/pages/Register.jsx [hunk 1]
```diff
@@ -52,6 +61,35 @@ const Resgister = () => {
       setPasswordError(null);
     }
 
+    if (!hasError) {
+      Api.register(firstname, lastname, username, password, {
+        firstname,
+        lastname,
+        username,
+        password,
```
Comment (ivan-clark) [inline frontend/src/pages/Register.jsx:69]: nagka baylo ang order sa mga parameters: maong inig ka register ang firstname nahimong username then ang lastname nahimong password

**Review by ivan-clark — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Register.jsx (hunk 1 above)
Comment (ivan-clark) [reply frontend/src/pages/Register.jsx:69]: Then ang usage ani is only: `Api.register(firstname, lastname, username, password)`

no need na ni ```
{
        firstname,
        lastname,
        username,
        password,
}
```

**Review by ivan-clark — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Register.jsx (hunk 1 above)
Comment (sulfur132) [reply frontend/src/pages/Register.jsx:69]: Already Fixed sirs.

**Review by ivan-clark — APPROVED** [review]
nice!

**Pull request merged**